Source/WebKit/UIProcess/mac/WKImmediateActionController.mm

```diff
@@ -487,12 +487,12 @@ - (NSSize)menuItem:(NSMenuItem *)menuItem maxSizeForPoint:(NSPoint)point
     if (!dictionaryPopupInfo.platformData.attributedString.nsAttributedString())
         return nil;
 
-    RetainPtr { _viewImpl.get() }->prepareForDictionaryLookup();
+    _viewImpl->prepareForDictionaryLookup();
```

Comment:
Oh, I think we want to create a copy of CheckedPtr here since _viewImpl can be cleared as a side effect of calling this function (not that it actually does but it's not obvious that'll never happen)